### ISSUE TITLE
Progressbar never completes after actions completed

### ISSUE THREAD

**Issue by chummels** [yt-project/yt#3209]
### Bug report

The yt progressbar that shows progress on a particular task never makes it to 100% even when actions have been completed.  It certainly used to, but at some point after a recent upgrade, it only reaches the penultimate progress status (90% below) before moving on.  It's not a huge deal, but it seems like it should be straightforward to fix.

**Code for reproduction**

```python
import yt
ds = yt.load('FIRE_M12i_ref11/snapshot_600.hdf5')
ds.index
```

**Output**

```
Initializing coarse index :  90%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▏             | 9/10 [00:00<00:00, 30.78it/s]
Initializing refined index:  90%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▏             | 9/10 [00:12<00:01,  1.34s/it]
```

**Comment by neutrinoceros** [comment]
So... this was originally solved by @brittonsmith in #3116
however, this is _also_ fixed in #3114.
I was probably a tad optimistic when I wrote that one and thought it would be discussed/accepted/merged much sooner, and I asked Britton if he was okay with closing his fix to save myself some conflict solving.
We should discuss this at next triage. Now I'm inclined to think that if #3114 (and the associated YTEP) can't receive the necessary attention to get merged reasnoably soon we should revived Briton's PR.

**Comment by munkm** [comment]
I definitely think yt-project/yt#3114 won't be able to get merged before yt 4.0, so we should try to fix this before the release.

**Comment by neutrinoceros** [comment]
Well that's a shame, but then #3116 has the minimal fix already.

**Comment by neutrinoceros** [comment]
fixed via #3116

**Comment by chummels** [comment]
Thanks, team!
